### PR TITLE
Remove data providers search

### PR DESCRIPTION
**Pull request by Joozty**
Should be merged after we release new search page for data providers

### CODE REVIEW

**Comment by Joozty** [comment]
@viktor-yakubiv I would probably suggest to remove /data/providers page and redirect it with 301 to the /data-providers?

File: data/documentation/fastsync.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note that this is an extremely large file (&Tilde;322GB) and appropriate tools a
 
 4.  The previous steps will extract the big archive in multiple smaller files.
     Each archive contains all the resources for a specific CORE data provider
-    the full list which you can find at our [data providers](/data/providers)
+    the full list which you can find at our [data providers](/data-providers)
```
Comment (viktor-yakubiv) [inline data/documentation/fastsync.md:32]: Hmm, technically this should have been `~data-providers` and did not require change but this is also fine with me.

**Review by viktor-yakubiv — APPROVED** [review]
I like how it's done. It's very good 👍

Thank you!

**Comment by viktor-yakubiv** [comment]
Thanks, I forgot about this

**Pull request merged**